reference/pipeline/expressions/index.md

```diff
@@ -261,24 +261,27 @@ For example, `${#triggerResolvedArtifact("my-image")["reference"]}` might return
 A shortcut to look up the resolved artifact in execution trigger by its type. If multiple artifacts are found, only 1 will be returned.
 For example, `${#triggerResolvedArtifactByType("docker/image")["reference"]}` might return `gcr.io/spinnaker-marketplace/orca@sha256:b48dbe7d7cb580db8512e4687d31f3710185b08afcf3cb53c0203025f93f9091`.
 
-## Whitelisted Java classes
+## Allowed Java classes
 
-You can find the code which whitelists Java classes [here](https://github.com/spinnaker/orca/blob/6d0ba0bf8af5e06c5b405b8294f07e7a5a4c335a/orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/expressions/whitelisting/InstantiationTypeRestrictor.java#L26).
-The whitelisted classes are:
+You can find the code which restricts instantiable Java classes [here](https://github.com/spinnaker/kork/blob/8e414e21625219a58afece0a812f570340882294/kork-expressions/src/main/java/com/netflix/spinnaker/kork/expressions/whitelisting/InstantiationTypeRestrictor.java#L28).
+The allowed classes are:
```

Comment:
I updated the link to point to where the code currently lives, and also added three more classes that have been added to the code.